ui/nginx.conf.prod

```diff
@@ -0,0 +1,30 @@
+# nginx server for static assets. For directive documentation, see
+# http://nginx.org/en/docs/dirindex.html
+
+# Required - just leave the defaults for now.
+events {}
+
+http {
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+
+    # If we're running on App Engine, logs will appear on the Google Developer's
+    # Console when logged to this directory.
+    access_log /var/log/app_engine/app.log;
+    error_log /var/log/app_engine/app.log;
+
+    gzip on;
```

Comment:
I confirmed things work without these two lines, can you delete them?

Reply:
Done